packages/mjml-column/src/index.js

```diff
@@ -34,6 +34,12 @@ export default class MjColumn extends BodyComponent {
     const paddingSize =
       this.getShorthandAttrValue('padding', 'left') +
       this.getShorthandAttrValue('padding', 'right')
+      
+    const border =
+      this.getShorthandBorderValue('right') +
+      this.getShorthandBorderValue('left')
+
+    const allPaddings = paddingSize + border
```

Comment:
I think the naming is a bit ugly here, since it does basically the job of box-sizing:border-box maybe we could create some kind of helper that would get an object of attributes and output an object `{ box:  450, border: 20, padding: 20, totalWidth: 490 }`

So it can be tested properly ?

Reply:
okay i'll do something like this 👍